packages/penrose-core/src/engine/Optimizer.ts

```diff
@@ -294,23 +295,11 @@ export const step = (state: State, steps: number, evaluate = true) => {
       // TODO. In the original optimizer, we cheat by using the EP cond here, because the UO cond is sometimes too strong.
       if (unconstrainedConverged2(normGrad)) {
         optParams.optStatus.tag = "UnconstrainedConverged";
-        memoryManager.deleteExcept([]); // Clear allocated matrix, vector objects in L-BFGS params
+        eig.GC.flush(); // Clear allocated matrix, vector objects in L-BFGS params
```

Comment:
This seems to be the accepted way (https://bertrandbev.github.io/eigen-js), but since the prior library still relied on garbage collection afaik due to emscripten only emulating wasm, it may be different under the hood.